hooks/pre-commit

```diff
@@ -9,3 +9,5 @@ SCRIPTS_DIR=$UNIVERSE_DIR/scripts
 
 $SCRIPTS_DIR/build.sh
 
+# Fail if there are unstaged changes
+git diff --exit-code
```

Comment:
We can make it nicer later, or figure out a different way to update the index.  It's important for now that we ensure that PRs are not breaking it though.  The unfortunate consequence is that PRs have a high potential of needing a rebase in order to be accepted for now.